.bazelrc

```diff
@@ -32,8 +32,6 @@ build:fat --ios_multi_cpus=i386,x86_64,armv7,arm64 --fat_apk_cpu=x86,x86_64,arme
 # TODO: Enable `--copt -ggdb3`. Issues were encountered previously with this:
```

Comment:
Is this still relevant?

Reply:
I guess it could be, but I'll remove it for now and we can revisit if we feel like we're missing this info